admin-manual/installation/installation.rst

```diff
@@ -577,8 +472,285 @@ Each service have a configuration file in /etc/sysconfig/archivematica-packagena
+machine you have been installing on).
+
+2. New Storage Service User
+
+Create a new administrative user in the Storage service. The storage service 
+has its own set of users. In the User menu in the Administrative tab of the 
+storage service, add at least one administrative user, and modify the
+test user, to change the password at a minimum. After you have created 
+an administrative user, copy its API key to your clipboard.
+
+3. Test the dashboard 
+
+You can login to the Archivematica dashboard and finish the installation in a 
+web browser: http://localhost (again, use the IP address of the machine you 
+have been installing on). When prompted, enter the URL of the Storage Service,
+the name of the administrative user, and that user's API key.
+
+4. Register your installation for full Format Policy Registry interoperability.
+
+Follow the instructions in the web browser to complete the installation.
+
+.. _upgrade:
+
+Upgrade from Archivematica 1.5
+==============================
+
+Archivematica 1.5.x is available for Ubuntu 14.04 and Centos 7.x.  If you are
+running a version of Archivematica older than 1.5.0, you will need to upgrade
+your operating system from Ubuntu 12.04 to Ubuntu 14.04, and upgrade 
+Archiveamtica to 1.5.1 before following these instructions.
+
+* :ref:`Upgrade Ubuntu Package Install <upgrade-ubuntu>`
+* :ref:`Upgrade CentOS/Redhat Package Install <upgrade-centos>`
+
+While it is possible to upgrade a github based source install using ansible,
+these instructions do not cover that scenario.
+
+**Backup first**
+
+Before starting any upgrade procedure on a production system, it is prudent to
+back up your system.  If you are using a virtual machine, take a snapshot of it
+before making any changes.  Alternatively, back up the file systems being used
+by your system.  Exact procedures for updating will depend on your local 
+installation.   At a minimum you should make backups of:
+
+* the storage service sqlite database
+* the dashboard mysql database
+
+A simple example of backing up these two databases:
+
+.. code:: bash
+
+   sudo cp /var/archivematica/storage-service/storage.db ~/storage_db_backup.db
+   mysqldump -u root -p MCP > ~/am_backup.sql
+
+If you do not have a password set for the root user in mysql, you can take out
+the '-p' portion of that command. If there is a problem during the upgrade
+process, you can restore your mysql database from this backup and try the
+upgrade again.
+
+.. _upgrade-ubuntu:
+
+Upgrade on Ubuntu
+-----------------
+ 
+1. Update OS 
+
+.. code:: bash
+
+   sudo apt-get update && sudo apt-get upgrade
+
+2. Update pip
+
+This is used to install python dependencies for both the storage service and 
+the dashboard.  There is a _known issue: https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1658844 with the version of pip installed on 
+Ubuntu 14.04, which makes this step necessary.
+
+.. code:: bash
+
+   wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py
```

Comment:
@mamedin found that updating setuptools is enough for the installation to proceed without issues.

So, instead of installing get-pip.py, we should be doing:

sudo pip install -U setuptools

This also allows to finish an 1.5-am install without issue.